lib/events/complete.go

```diff
@@ -170,7 +170,12 @@ func (u *UploadCompleter) checkUploads(ctx context.Context) error {
 		if _, err := u.cfg.SessionTracker.GetSessionTracker(ctx, upload.SessionID.String()); err == nil {
 			continue
 		} else if !trace.IsNotFound(err) {
-			return trace.Wrap(err)
+			// Ignore access denied errors, which we may get if the auth
+			// server is v9.2.1 or earlier, since only node, proxy, and
+			// kube roles had permission to create session trackers.
+			if !trace.IsAccessDenied(err) {
```

Comment:
What happens if you get access denied and session tracker is not actually created? Will anything break down the road?

Reply:
In this case we are treating that access denied error as a not found error, which is expected since the server can't create/get session trackers.

Reply:
Sounds good!